global/meta/models.yml

```diff
@@ -2022,6 +2022,9 @@ motion:
     type: timestamp
     read_only: true
     restriction_mode: C
+  start_line_number:
+    type: number
+    restriction_mode: A
```

Comment:
add `minimum: 1`

Reply:
Add the minimum to the start_line_number